pkg/webui/console/containers/device-importer/index.js

```diff
@@ -128,15 +168,42 @@ export default class DeviceImporter extends Component {
   }
 
   @bind
-  handleCreationProgress(device) {
+  handleCreationSuccess(device) {
     this.appendToLog(device)
-    this.setState(({ devicesComplete }) => ({ devicesComplete: devicesComplete + 1 }))
+    this.setState(({ devicesProcessed }) => ({
+      devicesProcessed: devicesProcessed + 1,
+    }))
   }
 
   @bind
-  handleError(error) {
-    const json = JSON.stringify(error, null, 2)
-    this.setState(({ log }) => ({ error, status: 'error', log: `${log}\n${json}` }))
+  handleCreationError(error) {
+    this.logError(error)
+    const { convertedDevices, devicesProcessed } = this.state
+    const currentDevice = convertedDevices.indexOf !== -1 ? convertedDevices[devicesProcessed] : {}
```

Comment:
```suggestion
    const currentDevice = convertedDevices.indexOf(...) !== -1 ? convertedDevices[devicesProcessed] : {}
```
`indexOf` is a function